inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -30,26 +30,39 @@
 
 
 @experiments.over('experiment_name', '^119..')
+@utils.for_each_value
 def experiment_name(self, key, value):
-    """Name of experiment."""
-    value = utils.force_list(value)
-    self.setdefault('affiliation', [v.get("u") for v in value if v.get('u')])
+    """Short experiment name."""
+    if 'u' in value:
+        self.setdefault('affiliations', [])
+        if isinstance(value['u'], str):
+            self['affiliations'].append(value['u'])
+        if isinstance(value['u'], (tuple, list)):
+            self['affiliations'].extend(value['u'])
     return {
-        'experiment': [v.get("a") for v in value if v.get('a')],
-        'wwwlab': [v.get("u") for v in value if v.get('u')]
+        'source': value.get('9'),
+        'subtitle': value.get('b'),
+        'title': value.get('a'),
     }
 
 
-@experiments.over('title', '^245[10_][0_]')
+@experiments.over('titles', '^245[10_][0_]')
 @utils.for_each_value
-@utils.filter_values
-def title(self, key, value):
-    """Title Statement."""
-    self.setdefault('breadcrumb_title', value.get('a'))
+def titles(self, key, value):
+    """Long experiment name."""
+    if 'a' in value:
+        self.setdefault('breadcrumb_title', value['a'])
```

Comment:
Just minor thing. I think with @kaplun and @jmartinm we figured out the `breadcrumb_title` was not longer used anywhere. If so, we can remove all mentions of it.